tests/org.eclipse.n4js.hlc.integrationtests/verifier/verifications-test.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2016 NumberFour AG.
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the Eclipse Public License v1.0
+which accompanies this distribution, and is available at
+http://www.eclipse.org/legal/epl-v10.html
+
+Contributors:
+  NumberFour AG - Initial API and implementation
+-->
+<verifications xmlns="http://maven.apache.org/verifications/1.0.0"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation="http://maven.apache.org/verifications/1.0.0 http://maven.apache.org/xsd/verifications-1.0.0.xsd">
+  <files>
+  <!--  
+    <file>
+      <location>src/main/resources/file1.txt</location>
+    </file>
+    <file>
+      <location>src/main/resources/file2.txt</location>
+      <contains>aaaabbbb</contains>
+    </file>
+    <file>
+      <location>src/main/resources/file3.txt</location>
+      <exists>false</exists>
+    </file>
+  -->
```

Comment:
do we need this commented comment?

Reply:
I think this file is not needed. Remove it